src/Symfony/Component/Lock/Store/FlockStore.php

```diff
@@ -27,7 +30,7 @@
  * @author Romain Neutron <imprec@gmail.com>
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class FlockStore implements StoreInterface
+class FlockStore implements BlockingStoreInterface, PersistStoreInterface, StoreInterface
```

Comment:
Same here, let's keep StoreInterface (the main one) as the first one listed here.